plugins/modules/aci_interface_policy_storm_control.py

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/python
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: aci_interface_policy_storm_control
+short_description: Manage Storm Control interface policies (stormctrl:IfPol)
+description:
+- Manage CDP interface policies on Cisco ACI fabrics.
+options:
+  stormctrl_policy:
+    description:
+    - The Storm Control interface policy name.
+    type: str
+    aliases: [ name ]
+  description:
+    description:
+    - The description for the Storm interface policy name.
+    type: str
+    aliases: [ descr ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+  rate:
```

Comment:
I think the input side of the rates could be improved. Would a per storm control type ( all, unicast, multicast, broadcast) with reusable argument specs ( dict definition in aci.py,, similar to like https://github.com/CiscoDevNet/ansible-aci/blob/master/plugins/module_utils/aci.py#L142 aci_owner_spec for instance) for the rates be an idea as input?

Input would then look something like below:
```
multicast ( same would apply for the other types ):
 - rate
 - burst_rate
 - type ( which would choice between percentage or packets per second)
```

This would also mean changing the logic of following parts of the code:

- the required definitions of AnsibleModule() init.
- the rates definition logic